### PR TITLE
remove fee type

### PR DESCRIPTION
**Pull request by sxlwar**
Darwinia runtime 27020

### CODE REVIEW

File: packages/types-known/src/spec/darwinia.ts [hunk 1]
```diff
@@ -91,6 +91,20 @@ const versioned: OverrideVersionedType[] = [
         moment: 'BlockNumber'
       }
     }
+  },
+  {
+    minmax: [1160, undefined],
+    types: {
+      ...sharedTypes,
+      AccountInfo: 'AccountInfoWithTripleRefCount',
```
Comment (acgxv) [inline packages/types-known/src/spec/darwinia.ts]: Why did we miss this before? And I think this is a native type of Polkadot.JS.

Comment (sxlwar) [reply packages/types-known/src/spec/darwinia.ts]: For debugging purposes,  add it back again. 

 We may not need it really @WoeOm 

https://github.com/polkadot-js/api/blob/43203195b48e944b6165430954f17fe3f0bea1e3/packages/types/src/interfaces/system/definitions.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L142-L170

Comment (WoeOm) [reply packages/types-known/src/spec/darwinia.ts]: 刚看 pangolin 也有 AccountInfoWithTripleRefCount ，pangolin 和 darwinia 类型不同吗？

apps 应该没有设置 AccountInfoWithTripleRefCount 为最新types

**Review by WoeOm — APPROVED** [review]

**Comment by WoeOm** [comment]
@chunqiuyiyu 
需要检查 apps 是否设置 AccountInfoWithTripleRefCount 为 AccountInfo 默认 type

**Comment by chunqiuyiyu** [comment]
> 

是
image

**Pull request merged**